packages/ra-input-rich-text/src/RichTextInput.tsx

```diff
@@ -108,10 +108,19 @@ export const RichTextInput = (props: RichTextInputProps) => {
 
     const { error, invalid, isTouched } = fieldState;
 
+    useEffect(() => {
+        if (!editor) return;
+
+        editor.commands.setContent(field.value, false, {
```

Comment:
I'm afraid this leads to an infinite query. Besides, this doesn't solve what the issue is about: changing the field value when the *record* changes.

Reply:
By passing `false` to the second argument of `editor.commands.setContent` command (emmitUpdate?: boolean), you prevent and infinite query.
I think it is a solution for the reported issue, with this change, the content of the `RichTextInput` updates correctly when the field value changes

Reply:
I've just added a test for it